frontend/src/features/admin-form/responses/InsightsPage/UnlockedInsights/FormChart.tsx

```diff
@@ -0,0 +1,129 @@
+import { useMemo, useState } from 'react'
+import Chart, { GoogleChartWrapperChartType } from 'react-google-charts'
+import { BiBarChartAlt2, BiTable } from 'react-icons/bi'
+import { Flex, Text, VStack } from '@chakra-ui/react'
+
+import { BasicField, FormFieldDto } from '~shared/types'
+
+import IconButton from '~components/IconButton'
+
+import { TableChart } from './TableChart'
+
+export const FIELD_TO_CHART = new Map<BasicField, GoogleChartWrapperChartType>([
+  [BasicField.Rating, 'ColumnChart'],
+  [BasicField.Radio, 'PieChart'],
+  [BasicField.Checkbox, 'BarChart'],
+  [BasicField.Dropdown, 'PieChart'],
+  [BasicField.CountryRegion, 'PieChart'],
+  [BasicField.YesNo, 'PieChart'],
+])
+
+export const FormChart = ({
+  title,
+  formField,
+  data,
+}: {
+  title: string
+  formField: FormFieldDto
+  data: [string, number | string][]
```

Comment:
[string, number | string][]  what is the string in the second parameter for?

Reply:
Its for the data header (`renderArray.unshift(['Answer', 'Count'])` +  color styling 😩